core/templates/dev/head/collection_editor/CollectionEditor.js

```diff
@@ -104,3 +104,61 @@ oppia.controller('CollectionEditor', ['$scope',
         });
     };
   }]);
+
+oppia.controller('CollectionEditorNavbarBreadcrumb', ['$scope',
+  'WritableCollectionBackendApiService', 'CollectionRightsBackendApiService',
+  'CollectionObjectFactory', 'SkillListObjectFactory',
+  'CollectionUpdateService', 'UndoRedoService', 'warningsData',
+  'routerService', function(
+    $scope, WritableCollectionBackendApiService,
+    CollectionRightsBackendApiService, CollectionObjectFactory,
+    SkillListObjectFactory, CollectionUpdateService, UndoRedoService,
+    warningsData, routerService) {
+    $scope.collection = null;
+    $scope.collectionId = GLOBALS.collectionId;
+    $scope.collectionSkillList = SkillListObjectFactory.create([]);
+    $scope.isPublic = GLOBALS.isPublic;
+
+    var _TAB_NAMES_TO_HUMAN_READABLE_NAMES = {
+      main: 'Edit',
+      preview: 'Preview',
+      settings: 'Settings',
+      stats: 'Statistics',
+      history: 'History',
+      feedback: 'Feedback'
+    };
+
+    WritableCollectionBackendApiService.fetchWritableCollection(
+      $scope.collectionId).then(
+        function(collectionBackendObject) {
+          $scope.collection = CollectionObjectFactory.create(
+            collectionBackendObject);
+          $scope.collectionSkillList.setSkills(collectionBackendObject.skills);
+        }, function(error) {
+          warningsData.addWarning(
+            error || 'There was an error loading the collection.');
+        });
+
+    $scope.getCurrentTabName = function() {
+      if (!routerService.getTabStatuses()) {
```

Comment:
It will not probably, since it always returns either 'main' or current tab status, so this check is redundant